modules/user/jwt.config.js

```diff
@@ -0,0 +1,17 @@
+const jwt = require("jsonwebtoken");
+require("dotenv").config();
```

Comment:
this dotenv init useless here, it should appear only once on application initialization